packages/volto/news/5924.bugfix

```diff
@@ -0,0 +1 @@
+Update volto.po "yes" translation si -> sì. @yurj
```

Comment:
```suggestion
Update italian locales volto.po "yes" translation si -> sì. @yurj
```

Reply:
I think it's better for us to mention which locales volto.po updates the yes translation as it might not be obvious for someone looking simply at the changelog which yes that belongs to